test-data/unit/daemon.test

```diff
@@ -138,3 +138,22 @@ mypy-daemon: error: Missing target module, package, files, or command.
 == Return code: 2
 $ dmypy stop
 Daemon stopped
+
+-- this is carefully constructed to be able to break if the quickstart system lets
+-- something through incorrectly. in particular, the files need to have the same size
+[case testDaemonQuickstart]
+$ {python} -c "print('x=1')" >foo.py
+$ mypy --local-partial-types --cache-fine-grained --follow-imports=error -- foo.py
+$ dmypy run -- foo.py --follow-imports=error --use-fine-grained-cache
+Daemon started
+$ dmypy status --fswatcher-dump-file test.json
+Daemon is up and running
+$ dmypy stop
+Daemon stopped
+-- sleep guarantees timestamp changes
+$ {python} -c "import time;time.sleep(1)"
+$ {python} -c "print('lol')" >foo.py
+$ dmypy run -- foo.py --follow-imports=error --use-fine-grained-cache --quickstart-file test.json
+Daemon started
+foo.py:1: error: Name 'lol' is not defined
+== Return code: 1
```

Comment:
So this test doesn't check that the quickstart does anything, right? It just checks that the behavior is the same.

Reply:
Yeah. I've updated the test to be able to make sure that a matching file doesn't have its metadata updated, which is a decent proxy for the thing we really care about